### ISSUE TITLE
how to make sure the test runs on a gpu

### ISSUE THREAD

**Issue by joepareti54** [LukasHedegaard/pytorch-benchmark#12]
i do not see any cpu to gpu or gpu to cpu information in the output file. To run I just followed the instructions in readme
[output_torch2.txt](https://github.com/LukasHedegaard/pytorch-benchmark/files/11236297/output_torch2.txt)

**Comment by LukasHedegaard** [comment]
Hi @joepareti54, 
The library transfers between CPU and whatever device the model you passed currently resides on. If the model you pass in is on "gpu0", this is the device that will be used. If the model is on "cpu", then no transfer occurs.
See [this line](https://github.com/LukasHedegaard/pytorch-benchmark/blob/e2e148109fedf82e64e50232b3421c95021f9ef5/pytorch_benchmark/benchmark.py#L79).

**Comment by joepareti54** [comment]
it seems to work on a gpu:
`import torch
from torchvision.models import efficientnet_b0
from pytorch_benchmark import benchmark
device_jp = torch.device("cuda" if torch.cuda.is_available() else "cpu")
model = efficientnet_b0()
sample = torch.randn(8, 3, 224, 224)  # (B, C, H, W)
model = model.to(device_jp)
results = benchmark(model, sample, num_runs=100)
print(results)`

**Comment by LukasHedegaard** [comment]
Indeed, this should be more in the README. I've updated the example with a comment as well made a note in the paragraph right before "Advanced use". Closing this issue for now.
